### PR TITLE
Simple new env for move-pick-place Spot Task!

### PR DESCRIPTION
**Pull request by NishanthJKumar**
Example command: `python predicators/main.py --env realworld_spot --approach oracle --seed 0 --num_train_tasks 0 --num_test_tasks 1 --spot_robot_ip IP`

### CODE REVIEW

File: predicators/envs/spot_env.py [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    pick objects from on top these surfaces, and then place them
+    elsewhere.
+    """
+
+    def __init__(self, use_gui: bool = True) -> None:
+        super().__init__(use_gui)
+
+        # Types
+        self._robot_type = Type("robot", [])
+        self._can_type = Type("soda_can", [])
```
Comment (wmcclinton) [inline predicators/envs/spot_env.py:33]: We might want to switch this to a general object type, and split into more specific ones later.

File: predicators/ml_models.py [hunk 2]
```diff
@@ -623,7 +623,6 @@ def _initialize_net(self) -> None:
         self._linears.append(nn.Linear(self._hid_sizes[-1], 1))
 
     def _create_loss_fn(self) -> Callable[[Tensor, Tensor], Tensor]:
-
```
Comment (wmcclinton) [inline predicators/ml_models.py:626]: revert

File: tests/envs/test_pybullet_cover.py [hunk 3]
```diff
@@ -13,7 +13,6 @@
 
 
 class _ExposedPyBulletCoverEnv(PyBulletCoverEnv):
-
```
Comment (wmcclinton) [inline tests/envs/test_pybullet_cover.py:16]: revert

**Review by wmcclinton — APPROVED** [review]
Good enough as a starting point! One thing we should figure out is what is the best way to deal with the graphs. Pushing and updating github every time is probably not be the best way.

File: predicators/envs/spot_env.py (hunk 1 above)
Comment (nkumar-bdai) [reply predicators/envs/spot_env.py:33]: i think let's leave it like this for now, since the predicates and operators would get more complicated if things were untyped
